### ISSUE TITLE
README file lists incorrect configuration option

### ISSUE THREAD

**Issue by daj** [KeepSafe/dexcount-gradle-plugin#33]
I tried configuring the dexcount plugin using the options described in the README file:
```
dexcount {
    includeClasses = false
    includeFieldCount = false
    includeTotalMethodCount = true
    printAsTree = false
    orderByMethodCount = true
    verbose = false
}
```
Unfortunately my build fails with this error:
```
* What went wrong:
A problem occurred evaluating project ':app'.
> No such property: includeMethodCount for class: com.getkeepsafe.dexcount.DexMethodCountExtension_Decorated
```
I tried changing `includeTotalMethodCount` to `includeMethodCount` and that didn't help either.

For now I've just commented out that configuration option so my build passes (obviously you can choose to not define the `dexcount` configuration block and use the defaults - that works fine too).

**Comment by barykaed** [comment]
+1 ... Please resolve.. 
Is this fixed on the snapshot releases?

**Comment by cesards** [comment]
+1 here.

**Comment by benjamin-bader** [comment]
I'm on vacation, will fix soon.  As noted, you can just remove the option and it will work as expected.

**Comment by benjamin-bader** [comment]
This should be resolved with the release of 0.3.1, which includes the documented config option.
